src/content/docs/features/ai-toolbox.md

```diff
@@ -59,6 +59,14 @@ When entering the `/help` command, you will see your custom command in the list
 
 ## Customizable System Prompt
 
+In Refact, a **system prompt** is used to maintain a standard format and clarity in communication. By default, it is set to:
+
+```
+You are a programming assistant. Use backquotes for code blocks, give links to documentation at the end of the response.
```

Comment:
I'm not sure that you should mention this prompt here, it can be changed
I propose to give an example of default system prompt customization in yaml format as given below